app/src/main/java/com/digitalarchitects/rmc_app/components/CommonUI.kt

```diff
@@ -832,3 +834,28 @@ fun RmcTopButton() {
     }
 }
 
+@Composable
+fun BooleanToggle(
```

Comment:
Dit is eigenlijk een Switch volgens Material Design. Ik zou het de RmcSwitch genoemd hebben.

Reply:
ja mee eens, ik pas het aan